src/types/index.ts

```diff
@@ -388,3 +388,61 @@ export type PaymentMethodResultType =
       paymentMethod: undefined;
       error: StripeError;
     };
+
+export type CollectInputsParameters = {
+  collectInputs: Input[];
+};
+
+export type CollectInputsResults = {
+  collectInputResults?: CollectInputResult[];
+  error: StripeError;
+};
+
+export type Input = {
+  inputType: 'EMAIL' | 'NUMERIC' | 'PHONE' | 'TEXT' | 'SELECTION' | 'SIGNATURE';
+  description?: string;
+  required: boolean;
+  skipButtonText?: string;
+  submitButtonText?: string;
+  title: string;
+  toggles?: Toggle[];
+  selectionButtons?: SelectionButton[];
+};
+
+export type CollectInputResult = {
+  skipped: boolean;
+  email?: string;
+  numericString?: string;
+  phone?: string;
+  selection?: string;
+  signatureSvg?: string;
+  text?: string;
+  toggles?: ToggleResult[];
+};
+
+export type Toggle = {
+  title: string;
+  description: string;
+  defaultValue: ToggleValue;
+};
+
+export enum ToggleValue {
+  ENABLED = 'ENABLED',
+  DISABLED = 'DISABLED',
+}
+
+export enum ToggleResult {
+  ENABLED = 'ENABLED',
+  DISABLED = 'DISABLED',
+  SKIPPED = 'SKIPPED',
+}
+
```

Comment:
@nazli-stripe @ianlin-bbpos Currently, the Android SDK supports toggles but not the iOS SDK.

I think we should remove these external models because both Android and iOS SDK don't support toggles. Or, if it's possible to include these models only in the Android SDK, I think we should leave it in and add code to transform the toggles in android/src/main/java/com/stripeterminalreactnative/StripeTerminalReactNativeModule.kt